test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -739,10 +739,11 @@ TEST_F(GrpcJsonTranscoderFilterTest, TranscodingUnaryWithHttpBodyAsOutputAndSpli
   EXPECT_EQ(Http::FilterTrailersStatus::Continue, filter_.decodeTrailers(response_trailers));
 }
 
-class GrpcJsonTranscoderFilterConvertGrpcStatusTest : public GrpcJsonTranscoderFilterTest {
+class GrpcJsonTranscoderFilterConvertGrpcStatusTestBase : public GrpcJsonTranscoderFilterTest {
```

Comment:
This doesn't do what describe in classname? i.e. `convert_grpc_status` is default false and that's not set here.